api/v1alpha1/moduletemplate_types.go

```diff
@@ -66,7 +66,7 @@ type ModuleTemplateSpec struct {
 
 	// Target describes where the Module should later on be installed if parsed correctly. It is used as installation
 	// hint by downstream controllers to determine which client implementation to use for working with the Module
-	Target Target `json:"target"`
+	Target Target `json:"target,omitempty"`
```

Comment:
if this is optional, what is the default? Maybe add that to the description

Reply:
without mentioning this target, the default is local single cluster mode, or should we in progress this issue to introduce a new target https://github.com/kyma-project/lifecycle-manager/issues/167

Reply:
But then how about the single cluster mode? should we immediately introduce new target https://github.com/kyma-project/lifecycle-manager/issues/167

Reply:
I am really not sure why we need an additional target right now. We have 2 targets already, control-plane and remote. Why is there a new target necessary?

Reply:
after discussion, the decision is keep the current target, make this field as mandatory, for single cluster mode, use `control-plane` as target, if have further request, extend the target definition.